addons/project/models/project_task.py

```diff
@@ -938,13 +939,40 @@ def create(self, vals_list):
             # if the portal user could really create the tasks based on the ir rule.
             tasks.with_user(self.env.user).check_access_rule('create')
         current_partner = self.env.user.partner_id
+
+        all_partner_emails = []
+        for task in tasks:
+            all_partner_emails += tools.email_split(task.email_cc)
+        partners = self.env['res.partner'].search([('email', 'in', all_partner_emails)])
+        partner_per_email = {
+            partner.email: partner
+            for partner in partners
+            if not all(u.share for u in partner.user_ids)
+        }
         for task in tasks:
             if task.project_id.privacy_visibility == 'portal':
                 task._portal_ensure_token()
             for follower in task.parent_id.message_follower_ids:
                 task.message_subscribe(follower.partner_id.ids, follower.subtype_ids.ids)
             if current_partner not in task.message_partner_ids:
                 task.message_subscribe(current_partner.ids)
+            if task.email_cc:
+                partners_with_internal_user_ids = [
+                    partner_per_email[email].id
+                    for email in tools.email_split(task.email_cc)
+                    if partner_per_email.get(email)
+                ]
+                if not partners_with_internal_user_ids:
+                    continue
+                task.message_subscribe(partners_with_internal_user_ids)
+                task.message_notify(
+                    subject=Markup(_('Invitation to follow Task: %(task_name)s') % {'task_name': task.name}),
```

Comment:
Do you actually need a markup here?
```suggestion
                    subject=_('Invitation to follow Task: %(task_name)s', task_name=task.name),
```

Reply:
If i understood correctly the explanation of xmo, and the link he gave, with the following task name 'problem with the tag <p>', the '<p>' will be escape without the markup and thus will not appear in the task name, no ?

Reply:
actually, can't you use the `portal_share_template`?

Reply:
I did not know it existed :sweat_smile: . I took a quick glance, and indeed it looks appropriate. I'll try to use it and see how it goes. Thanks for your comment.